### PR TITLE
Split out Mongo params.

### PR DESCRIPTION
**Pull request by derrickburns**

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -eu
```
Comment (pazaan) [inline test]: Is this file needed? Looks like a duplicate of `test.sh`.

**Comment by derrickburns** [comment]
@pazaan fixed.
